logger/event_logger.go

```diff
@@ -1,49 +1,60 @@
+//+build windows
+
 package logger
 
 import (
+	"fmt"
 	"io"
+	"log"
 	"strings"
 
 	"github.com/influxdata/wlog"
-	"github.com/kardianos/service"
+	"golang.org/x/sys/windows/svc/eventlog"
 )
 
 const (
 	LogTargetEventlog = "eventlog"
 )
 
 type eventLogger struct {
-	logger service.Logger
+	logger *eventlog.Log
 }
 
 func (t *eventLogger) Write(b []byte) (n int, err error) {
 	loc := prefixRegex.FindIndex(b)
 	n = len(b)
 	if loc == nil {
-		err = t.logger.Info(b)
+		err = t.logger.Info(1, fmt.Sprint(b))
 	} else if n > 2 { //skip empty log messages
 		line := strings.Trim(string(b[loc[1]:]), " \t\r\n")
 		switch rune(b[loc[0]]) {
 		case 'I':
-			err = t.logger.Info(line)
+			err = t.logger.Info(1, line)
```

Comment:
what are the numbers here?

Reply:
Something called eid, the package we were using before this was calling these methods that require the 'eid' itself, and putting 1 for info, 2 for warning, and 3 for error. I'm calling it directly here because I had to ditch that package to make this work

Reply:
could make consts for it.. eidInfo, eidWarn, eidError

Reply:
updated.